example/plugin-system/index.html

```diff
@@ -26,7 +26,7 @@
         <script src="../../dist/plugin/wavesurfer.spectrogram.js"></script>
 
         <!-- App -->
-        <script src="app.js"></script>
+        <script type="module" src="app.js"></script>
```

Comment:
is this necessary?

Reply:
Yes, I added a motivation in description